nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -54,6 +54,8 @@
 
 - `buildbot` was updated to 4.0, the AngularJS frontend has been replaced by a React frontend, see the [upstream release notes](https://docs.buildbot.net/current/manual/upgrading/4.0-upgrade.html).
 
+- `jre_minimal.tests` is now an attribute set of package tests instead of a list, making it easier to access individual test cases. This change does not affect the CI, since `nix-build jre_minimal.tests` still works.
```

Comment:
I don't think this is noteworthy at all.

Reply:
I had thought it would be considered breaking.

Reply:
Addressed.